### PR TITLE
DOCSP-42256: leftover v2 migration items

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42256>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-42256-v2-mig-items/whats-new/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -122,6 +122,13 @@ The 2.0 {+driver-short+} release includes the following improvements and fixes:
   ``SessionOptions`` type. To learn more, see the
   :ref:`golang-session-txn-options` section of the Transactions guide.
 
+- The ``Cursor.SetMaxTime()`` method is renamed to
+  ``Cursor.SetMaxAwaitTime()``. This method specifies the maximum time
+  that the server waits for new documents when you retrieve documents by
+  using a tailable cursor when querying a capped collection.
```
Comment (jordan-smith721) [inline source/whats-new.txt:128]: S:  This is a pretty wordy sentence. Can it be simplified to something like:
```suggestion
  ``Cursor.SetMaxAwaitTime()``. This method specifies the maximum time
  that the server waits for new documents when retrieving documents with a tailable cursor on a capped collection.
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -122,6 +122,13 @@ The 2.0 {+driver-short+} release includes the following improvements and fixes:
   ``SessionOptions`` type. To learn more, see the
   :ref:`golang-session-txn-options` section of the Transactions guide.
 
+- The ``Cursor.SetMaxTime()`` method is renamed to
```
Comment (jordan-smith721) [inline source/whats-new.txt:125]: Q: Is this a breaking change since it changes a method name?

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ a question and suggestion

**Comment by rustagir** [comment]
@prestonvasquez reminder for you to review!

**Review by prestonvasquez — APPROVED** [review]

**Pull request merged**